platform/cluster/saltbase/salt/kubelet/kubelet-healthcheck

```diff
@@ -30,12 +28,16 @@ instance-id-to-tags () {
 
 dump-system () {
     echo "===== ${KUBELET_HC_LOG_TAG} log =====" >> ${KUBELET_HC_LOG_PATH}
-    grep -F ${KUBELET_HC_LOG_TAG} /var/log/syslog >> ${KUBELET_HC_LOG_PATH}
+    grep -F "${KUBELET_HC_LOG_TAG}" /var/log/syslog >> ${KUBELET_HC_LOG_PATH}
 
     echo >> ${KUBELET_HC_LOG_PATH}
     echo >> ${KUBELET_HC_LOG_PATH}
-    echo "===== Current CNI Information =====" >> ${KUBELET_HC_LOG_PATH}
-    ls -l ${KUBELET_CNI_DIR} >> ${KUBELET_HC_LOG_PATH} 2>&1
+    echo >> ${KUBELET_HC_LOG_PATH}
+    echo >> ${KUBELET_HC_LOG_PATH}
+    echo >> ${KUBELET_HC_LOG_PATH}
+
+    echo "===== Last 20000 lines of kubelet log =====" >> ${KUBELET_HC_LOG_PATH}
```

Comment:
This could be empty after a log rotate.

Reply:
yes, best effort for now, same for docker healthcheck